src/slackbot/check-results-labeled.ts

```diff
@@ -42,7 +42,12 @@ export async function summarizeCheckResultsToLabeledCheckStatus(
       return acc;
     }, new Map<string, { meanPassRate: number; passRateStdDev: number }>());
 
-  log.debug("MEAN PASS RATE\n" + meanPassRatePerCheckAndLocation.toString());
+  log.debug(
```

Comment:
tip: you may want to look for a debugger, it may be more convenient than dumping objects into logs